docker/runtime-config-template.json

```diff
@@ -1,20 +1,17 @@
 {
     "backendUrl": "${REACT_APP_BACKEND_URL}",
     "backendWSUrl": "${REACT_APP_BACKEND_WS_URL}",
-    "enableWebsocketSupport": ${REACT_APP_ENABLE_WEBSOCKETS_SUPPORT},
-    "useSecureCookie": ${REACT_APP_USE_SECURE_COOKIE},
-    "debug": ${REACT_APP_DEBUG},
-    "cookieDomain": "localhost",
-
-    "defaultAutoRefreshInterval": ${REACT_APP_DEFAULT_AUTO_REFRESH_INTERVAL},
-    "realtimeServiceMaxRetries": ${REACT_APP_REALTIME_SERVICE_MAX_RETRIES},
-
+    "enableWebsocketSupport": "${REACT_APP_ENABLE_WEBSOCKETS_SUPPORT}",
+    "useSecureCookie": "${REACT_APP_USE_SECURE_COOKIE}",
+    "debug": "${REACT_APP_DEBUG}",
+    "cookieDomain": "${REACT_APP_USE_SECURE_COOKIE}",
```

Comment:
Question: These two variables are supposed to be booleans, but this forces them to JSON strings.  I see this is handled if reading directly from the environment under npm, but how is this handled when reading `runtime-config.json` directly?

Reply:
Sorry, I did not intend to flag `cookieDomain` here, but rather `enableWebsocketSupport` (in addition to `debug`)